webpack.config.js

```diff
@@ -185,16 +185,30 @@ var config = {
     '#cheap-module-eval-source-map'
 };
 
+
+function hasZopfli() {
```

Comment:
> I'll replicate this to sentry-plugins too.

For some reason, it's not explicitly used for getsentry, so it doesn't need to go there, but we should probably add this snippet anyways to precompress there.